build/common.rs

```diff
@@ -43,6 +43,7 @@ const ALL_COMPONENTS: &[&str] = &[
     "comp_esp_timer_enabled",
     "comp_esp_tls_enabled",
     "comp_esp_wifi_enabled",
+    "comp_esp_espnow_enabled",
```

Comment:
I don't see a component (i.e. a directory) named `espnow` [here](https://github.com/espressif/esp-idf/tree/master/components). Are you sure such a component exists, and `espnow` is not simply a part of the `esp_wifi` component?

Reply:
the header file seems to be living here https://github.com/espressif/esp-idf/blob/master/components/esp_wifi/include/esp_now.h

Ah! I was not sure what `comp` stands for here - if it's 1:1 mapping with dirs then the 'flag' should be probably removed and the header should get included as a part of wifi coponent